google-cloud-testing/google-cloud-conformance-tests/pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-conformance-tests</artifactId>
+  <!-- TODO(benwhitehead): What should this version actually be? -->
+  <version>0.1.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-conformance-tests:current} -->
```

Comment:
Since this is only used for the tests of clients I don't think it needs to go to Maven Central. I'll remove the replace tag here.

Is there anything else I need to add into the pom to exclude it from publishing?

Reply:
The parent is already excluding the modules under `google-cloud-testing` from being staged/released.